src/Microsoft.Framework.PackageManager/Publish/PublishRuntime.cs

```diff
@@ -11,11 +11,13 @@ public class PublishRuntime
     {
         private readonly FrameworkName _frameworkName;
         private readonly string _runtimePath;
+        private readonly bool _isMono;
 
-        public PublishRuntime(PublishRoot root, FrameworkName frameworkName, string runtimePath)
+        public PublishRuntime(PublishRoot root, FrameworkName frameworkName, string runtimePath, bool isMono)
```

Comment:
I guess it's better to have `PublishRoot.IsMono` instead of passing it in this way.